fission-web-api/library/Fission/Web/API/User/Email/Types.hs

```diff
@@ -4,4 +4,10 @@ import           Fission.Web.API.Prelude
 
 import           Fission.Web.API.User.Email.Verify.Types
 
-type Email = "email" :> Verify
+import           Fission.Web.API.User.Email.Resend.Types
+
+type Email = "email" :> API
+
+type API
+  =    Verify
+  :<|> Resend
```

Comment:
To keep things totally type safe (see comment below), we should probably do something along these lines:

```haskell
type Email
  =    Verify'
  :<|> Resend'

type Prefix = "email"

type Verify'   = Prefix :> Verify
type Resend' = Prefix :> Resend
```

... plus an exported prefix for `User`.

Thoughts?

Reply:
Re: Type safety: I don't think there's a type safety issue. If I write
```hs
route = Proxy @("usser" :> "somethingwrong" :> API.User.Email.Verify)
```
The code type for `API.mkLink` prevents this from compiling:
```haskell
mkLink :: (IsElem route Fission.API, HasLink route) => Proxy route -> MkLink route Link
```

Reply:
Excellent point! I knew we set all this stuff up for a reason 😅